src/xai_grok/schemas/chat_response.py

```diff
@@ -0,0 +1,17 @@
+import datetime
+import typing
+
+import pydantic
+
+from xai_grok.schemas.choice import Choice
+from xai_grok.schemas.usage import Usage
+
+
+class ChatResponse(pydantic.BaseModel):
+    id: str
+    object: str
+    created: datetime.datetime
+    model: str
```

Comment:
The `created` field uses `datetime.datetime`, which may not be JSON serializable. Consider using `pydantic`'s `datetime` type for automatic serialization.